app/views/govuk_publishing_components/components/docs/details.yml

```diff
@@ -22,6 +22,18 @@ examples:
       block: |
         We need to know your nationality so we can work out which elections you’re entitled to vote in. If you can’t provide your nationality, you’ll have to send copies of identity documents through the post.
   with_data_attributes:
+    description: Can be used for tracking. By default, `track-label` is set to the status ("open" or "closed") unless a track_label is passed into the component.
+    data:
+      title: Help with nationality
+      data_attributes:
+        track_category: "track-category"
```

Comment:
The umbrella term eg `brexitCheckerHelper` Something that identifies all helper messages with a specific feature

Reply:
@huwd I don't want to make the example too specific to the Brexit checker, as it's really an analytics pattern decided by PAs rather than the component itself. But I've used actual examples now rather than placeholder strings.